### ISSUE TITLE
font size hackery

### ISSUE THREAD

**Issue by chad3814** [dupontgu/now-playing-bumper-sticker#1]
I think if you used the [skia canvas](https://deno.land/x/skia_canvas@0.5.6) you wouldn't have to do the [hackery](https://github.com/dupontgu/now-playing-bumper-sticker/blob/main/valtown/bumper_sticker_render.ts#L8). It should support `measureText()` with the custom loaded font.

**Comment by dupontgu** [comment]
ah nice, I'll try it out. The implementation I used also has `measureText`, but it only seems to work accurately on monospaced typefaces. I have a slight hunch I'll run into the same issue but definitely worth a shot - thx for the tip!

**Comment by chad3814** [comment]
yeah, I was trying to decode and figure out why it wasn't using the loaded font, but that deno `canvas` you're currently using uses a wasm of the skia canvas from some specific date, and it's been run through webpack, so it's uglified. fwiw, I don't know anything about deno, but I have used the `node-canvas` extensively, and specifically with `measureText()` with custom fonts.
